src/Presentation/DonorDataFormatter.php

```diff
@@ -9,6 +9,9 @@
 
 class DonorDataFormatter {
 
+	/**
+	 * @return array<string>|array<string, bool>
```

Comment:
```suggestion
	 * @return array<string, scalar>
```

shows the intent of this better